com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/LuaBuilder.java

```diff
@@ -62,6 +65,36 @@
 
     private static ArrayList<Platform> needsLuaSource = new ArrayList<Platform>(Arrays.asList(Platform.JsWeb, Platform.WasmWeb));
 
+    private static List<LuaBuilderPlugin> luaBuilderPlugins = null;
+
+    private void initBuilderPlugins() throws CompileExceptionError{
+        if (luaBuilderPlugins != null) {
+            return;
+        }
+
+        luaBuilderPlugins = new ArrayList<LuaBuilderPlugin>();
+        IClassScanner scanner = Project.getClassLoaderScanner();
+        Set<String> classNames = scanner.scan("com.dynamo.bob.pipeline");
```

Comment:
Currently a requirement that any plugins are in the com.dynamo.bob.pipeline package